.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: Document workspace
         env:
           RUSTDOCFLAGS: "--cfg docsrs"
-        run: cargo doc --features static,zlib,blosc,lzf
+        run: cargo doc --features static,zlib,blosc,lzf,f16,complex
```

Comment:
These flags should be added to `package.metadata.docs.rs` in `Cargo.toml`, both for the `hdf5` package and the `hdf5-types` (which lacks this entry).

Reply:
Yea, good point - added them to both.